### PR TITLE
Web branch9112017

### PR DESCRIPTION
**Pull request by danielzar2710**
All the design changes + sort& filter

### CODE REVIEW

**Comment by NajibAbuSamra** [comment]
are these the same changes as before? or a new change?

**Comment by danielzar2710** [comment]
what was before and i added the possiblity to ruti to sort and filter her tables

File: WebContent/WEB-INF/css/styles.css [hunk 1]
```diff
@@ -239,18 +277,19 @@ div.bottomPart{
 	width:100%;
 }
 div.inner{
-border-radius: 20px;
+	border-radius: 20px;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css:280]: nit: chnage tab to spaces

File: WebContent/WEB-INF/css/styles.css [hunk 2]
```diff
@@ -762,4 +800,79 @@ a.reports
   87.5% {
     box-shadow: 0em -3em 0 0, 2em -2em 0 -1em, 3em 0 0 -1em, 2em 2em 0 -1em, 0 3em 0 -1em, -2em 2em 0 0, -3em 0em 0 0, -2em -2em 0 0.2em;
   }
+}
+.DannyModal{
+	position: fixed;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css]: nit: indent

File: WebContent/WEB-INF/css/styles.css [hunk 3]
```diff
@@ -762,4 +800,79 @@ a.reports
...
+    right: 0;
+    bottom: 0;
+    left: 0;
+    width: 35%;
+    height: 22%;
+    background-color: white;
+}
+footer{
+	width:100%;
+    position: absolute;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css]: nit: indent

File: WebContent/WEB-INF/css/styles.css [hunk 4]
```diff
@@ -762,4 +800,79 @@ a.reports
...
+    margin-top: 4vh;
+	
+}
+.decision{
+	position:absolute;
+	height:100%;
+	width:50%;
+	
+	    display: block;
+    min-height: 60px;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css]: nit: indents

File: WebContent/WEB-INF/css/styles.css [hunk 5]
```diff
@@ -762,4 +800,79 @@ a.reports
...
+	text-align:center;
+	color:white;
+	font-family: Helvetica;
+	cursor: pointer;
+}
+.yes{
+	border-bottom-left-radius:12px;
+	left:0;
+	background-color: rgb(46,204,64);
+	    font-size: 3vh;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css]: nit: indent

File: WebContent/WEB-INF/css/styles.css [hunk 6]
```diff
@@ -762,4 +800,79 @@ a.reports
...
+}
+.no{
+	border-bottom-right-radius:12px;
+	right:0;
+	background-color:rgb(252,113,105);
+	    font-size: 3vh;
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/css/styles.css]: nit: indent

File: WebContent/WEB-INF/scripts/index.js [hunk 7]
```diff
@@ -21,6 +21,8 @@ $(document).ready(function(){
 			  select[i].childNodes[1].selected= true;
 		} 
 		  this.submit();
+		  var body=document.getElementsByTagName("body")[0];
+			body.innerHTML='<div class="waiting"><div class="loader">Loading...</div><p class="sorry">Sorry, Please wait...<br>We are making the world a better place one pair at a time :)</p>';
```
Comment (NajibAbuSamra) [inline WebContent/WEB-INF/scripts/index.js:25]: nit: indent

**Review by firdos94 — APPROVED** [review]

File: WebContent/mainPair.jsp [hunk 8]
```diff
@@ -92,8 +87,126 @@ function pairTableToArray(param)
...
+}
+function disconnect(param)
+{
+	$.post("DisconnectPair",
+	        {
+	          pairId: param,
+	        },
+	        function(data,status){
+	        	//if data is -1 something is wrong
+	            $("#"+data).parent().parent().remove();
```
Comment (NajibAbuSamra) [inline WebContent/mainPair.jsp:106]: no need to check status or data?

File: WebContent/WEB-INF/css/styles.css (hunk 2 above)
Comment (danielzar2710) [reply WebContent/WEB-INF/css/styles.css]: @NajibAbuSamra i do not see a problem here

**Comment by danielzar2710** [comment]
i have changed all the points

**Review by NajibAbuSamra — APPROVED** [review]

**Pull request merged**